app/assets/stylesheets/custom.css.scss

```diff
@@ -0,0 +1,92 @@
+.landing-page-body {
+  margin: 0 auto;
+  width: 100%;
+  padding-top: 0px !important;
+  min-height:100%;
+  background-color:#000000;
+  color:#eee;
```

Comment:
Color literals like `#eee` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Colon after property should be followed by one space